src/packaging/webots_distro.c

```diff
@@ -836,8 +836,6 @@ static void create_file(const char *name, int m) {
       fprintf(fd, "cd \"%s/%s/lib/webots\"\n", distribution_path, bundle_name);
       fprintf(fd, "ln -s libssl.1.0.0.dylib libssl.dylib\n");
       fprintf(fd, "ln -s libcrypto.1.0.0.dylib libcrypto.dylib\n");
-      fprintf(fd, "ln -s libssh.4.dylib libssh.dylib\n");
-      fprintf(fd, "ln -s libzip.2.dylib libzip.dylib\n");
```

Comment:
Don't we need these libraries for the Darwin-OP robot window?

Reply:
Oh yes, I see, they are installed locally...

Reply:
Exactly, actually here this command was just creating a symlink to a file not existing 😄